### PR TITLE
Migrate everything to TS

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -44,7 +44,6 @@ rules:
   no-multi-str: "off"
   no-unused-vars: "off"
   "@typescript-eslint/no-unused-vars": ["error", { "ignoreRestSiblings": true }]
-  "@typescript-eslint/explicit-module-boundary-types": "error"
```
Comment (sandrica89) [inline .eslintrc.yml:47]: Why remove it? It ensures the result in correct type, isn't it?

File: db/psychologists.ts [hunk 2]
```diff
@@ -104,7 +100,7 @@ const getById = async (psychologistId: string): Promise<Psychologist> => {
 };
 
 /**
- * Perform a UPSERT with https://knexjs.org/#Builder-merge
+ * Perform a UPSERT with https://dbjs.org/#Builder-merge
```
Comment (sandrica89) [inline db/psychologists.ts:103]: Refactor error :)

File: .eslintrc.yml (hunk 1 above)
Comment (desoindx) [reply .eslintrc.yml:47]: yes, but as i told you, i didn't migrate all the type yet => next PR

File: db/patients.ts [hunk 3]
```diff
@@ -62,12 +61,21 @@ module.exports.insert = async (
...
 
-module.exports.update = async (
-  id, firstNames, lastName, INE, institutionName, isStudentStatusVerified, hasPrescription, psychologistId, doctorName,
-  doctorAddress, dateOfBirth,
-) => {
+const update = async (
+  id: string,
+  firstNames: string,
+  lastName: string,
+  INE: string,
```
[the remark this answers is not in the document]
Comment (desoindx) [reply db/patients.ts:68]: yes => same as before, will be perfected on the pr for types

[the remark this answers is not in the document]
Comment (sandrica89) [reply db/patients.ts:68]: and keeping it like this until we add correct types won't cause any issue?

[the remark this answers is not in the document]
Comment (desoindx) [reply db/patients.ts:68]: no, TS is just declarative, at runtime nothing is done

**Review by sandrica89 — APPROVED** [review]

**Pull request merged**